core/src/com/unciv/logic/civilization/CivInfoStats.kt

```diff
@@ -108,6 +108,17 @@ class CivInfoStats(val civInfo: CivilizationInfo) {
         statMap["Transportation upkeep"] = Stats().apply { gold = -getTransportationUpkeep().toFloat() }
         statMap["Unit upkeep"] = Stats().apply { gold = -getUnitMaintenance().toFloat() }
 
+        if (civInfo.religionManager.religion != null) {
+            for (unique in civInfo.religionManager.religion!!.getFounderBeliefs().flatMap { it.uniqueObjects }) {
+                if (unique.placeholderText == "[] for each global city following this religion") {
+                    statMap.add("Religion", unique.stats.apply {
```

Comment:
unique.stats.times(...), you don *not* want to use timesInPlace because that will change the stats of the unique, which it will apply to the next city, so this will keep on growing...